firefox_ui_harness/runtests.py

```diff
@@ -16,7 +16,46 @@
 
 class ReleaseTestRunner(BaseMarionetteTestRunner):
     extra_prefs = {
-        "browser.tabs.remote.autostart": False,
+        'browser.tabs.remote.autostart': False,
+        'app.update.auto': False,
```

Comment:
Please make sure to sort all the preferences alphabetically. Beside that I would prefer if you could move out this big dictionary from the class into the global scope, or better into a separate file which we can import. Maybe default_prefs.py.